web/src/main/kotlin/com/example/ajaxproject/nats/GetUserByIdNatsController.kt

```diff
@@ -0,0 +1,39 @@
+package com.example.ajaxproject.nats
+
+import com.example.ajaxproject.GetByIdUserRequest
+import com.example.ajaxproject.GetByIdUserResponse
+import com.example.ajaxproject.NatsSubject.GET_USER_BY_ID
+import com.example.ajaxproject.User
+import com.example.ajaxproject.model.toProtoUser
+import com.example.ajaxproject.service.interfaces.UserService
+import com.google.protobuf.Parser
+import io.nats.client.Connection
+import org.springframework.stereotype.Component
+
+@Component
+class GetUserByIdNatsController(
+    override val connection: Connection,
+    private val userService: UserService,
+) : NatsController<GetByIdUserRequest, GetByIdUserResponse> {
+
+    override val subject = GET_USER_BY_ID
+    override val parser: Parser<GetByIdUserRequest> = GetByIdUserRequest.parser()
+
+    override fun handle(request: GetByIdUserRequest): GetByIdUserResponse = runCatching {
+        val userId = request.userId
+        val getUserById = userService.getUserById(userId)
```

Comment:
You're using userId only once, maybe this will be look's better:

```suggestion
        val getUserById = userService.getUserById(request.userId)
```